### PR TITLE
[1.24] Remove ServiceMeshPolicy and SMRbacConfig for Maistra installations

### PR DESCRIPTION
**Pull request by xeviknal**
cherry-pick of https://github.com/kiali/kiali-operator/pull/144

### CODE REVIEW

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
Same as the other PR - remove these from the v1.24 roles as well.

**Comment by xeviknal** [comment]
@jmazzitelli cherry-pick from other PR done.

**Review by jmazzitelli — APPROVED** [review]
LGTM

**Review by lucasponce — APPROVED** [review]

File: manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml [hunk 1]
```diff
@@ -521,26 +521,6 @@ spec:
...
-        - apiGroups: ["rbac.maistra.io"]
-          resources:
-          - servicemeshrbacconfigs
-          verbs:
-          - create
-          - delete
-          - get
-          - list
-          - patch
-          - watch
```
Comment (jmazzitelli) [inline manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml:543]: we need to revert this change to the ossm golden copy - in order to support installing the older 1.12 stuff, these permissions are still required.

Comment (jmazzitelli) [reply manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml:543]: revert PRs:
* https://github.com/kiali/kiali-operator/pull/159
* https://github.com/kiali/kiali-operator/pull/160

**Pull request merged**